### DISCUSSION TITLE
yolov5结合realsense D455实现目标检测

### DISCUSSION THREAD

**Discussion by zgr-LP** [community/community#84889] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

我想用yolov5结合realsense D455实现目标检测，但是相机反馈的视频帧率太低，我想到了用onnxruntime来对模型进行推理优化，从逻辑上怎么样把onnxruntime部署在模型上求解答

**Comment by Akash1134** [comment]
Hi @zgr-LP  , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
